index.js

```diff
@@ -99,13 +103,37 @@ var pcbStackup = function(layers, options, done) {
       layerOptions.plotAsOutline = options.outlineGapFill
     }
 
-    var converter = gerberToSvg(layer.gerber, layerOptions, finishLayer)
+    var converterDone = function() {
+      layer.done = true
+    }
+
+    var converter
+
+    if (layer.converter != null && layer.gerber == null) {
+      converter = layer.converter
+      converterDone()
+    }
+    else {
+      converter = gerberToSvg(layer.gerber, layerOptions, converterDone)
+    }
 
     stackupLayers.push({
       type: layerType,
       converter: converter,
       options: layerOptions
     })
+
+    var finishIfDone = function() {
```

Comment:
Valid point. If a gerber and a converter are both specified, I think I'd assume the gerber to take precedence. Can you think of a use case?